Rename Indexer client to Generator client throughout documentation (#253)
shinzonetwork/docs: content/reference/components/viewkit/index.md

  Host->>User: view documents
{% end %}

> Generators are not involved in the view lifecycle. By the time a view is created and applied, generator have already delivered raw data to hosts over P2P.

Reviewer: ```suggestion
Generator clients are not involved in the View lifecycle. By the time a View is created and applied, the Generator client has already delivered raw data to the Host clients over P2P.
```
